app/src/main/res/values/strings.xml

```diff
@@ -39,6 +39,8 @@
     <string name="session_unchecked">Unchecked this session</string>
     <string name="session_check">Check</string>
     <string name="session_uncheck">Uncheck</string>
+    <string name="my_session">My Session</string>
+    <string name="my_session_empty_description">You don\'t save a my session. To see my session, please you save it in session detail page.</string>
```

Comment:
Can you change this message to this one? 🙇 
https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/res/values/strings.xml#L110